test/ngTouch/directive/ngClickSpec.js

```diff
@@ -49,6 +49,16 @@ describe('ngClick (touch)', function() {
   }));
 
 
+  it('should unwrap a jQuery-wrapped event object', inject(function($rootScope, $compile) {
+    element = $compile('<div ng-click="event = $event"></div>')($rootScope);
+    $rootScope.$digest();
+
+    browserTrigger(element, 'touchstart');
+    browserTrigger(element, 'touchend');
+    expect($rootScope.event.originalEvent).toBeUndefined();
```

Comment:
BTW, this test will pass even if only `touchend` is patched. It'd be good to test them both.